tests/integration/mixer/telemetry/metrics/scenarios_test.go

```diff
@@ -201,6 +201,8 @@ values:
     enabled: true
   global:
     disablePolicyChecks: false
+    outboundTrafficPolicy:
+      mode: REGISTRY_ONLY
```

Comment:
Why do we need this?

Reply:
We don't. I most likely altered it by accident. Reverted.